src/main/scala/viper/gobra/frontend/info/implementation/typing/ExprTyping.scala

```diff
@@ -135,7 +135,7 @@ trait ExprTyping extends BaseTyping { this: TypeInfoImpl =>
       case (Right(_), Some(p: ap.Conversion)) => // requires single argument and the expression has to be convertible to target type
         val msgs = message(n, "expected a single argument", p.arg.size != 1)
         if (msgs.nonEmpty) msgs
-        else convertibleTo.errors(exprType(p.arg.head), typeType(p.typ))(n) ++ isExpr(p.arg.head).out
+        else convertibleTo.errors(typeType(p.typ), exprType(p.arg.head))(n) ++ isExpr(p.arg.head).out
```

Comment:
The order seems wrong to me. The type of the expression should be convertable to the argument type.

Reply:
I believe you are right, I miss-interpreted the property signature